myuw/templates/500.html

```diff
@@ -101,53 +150,62 @@
-         <!-- footer -->
-    <div id="app_footer">
-        <div class="container navigation-bar" role="menu" aria-labeledby="categories">
-            <h2 id="categories" class="sr-only">Resources</h2>
-            <ul class="clearfix navigation-list">
-                <li class="pull-left">
-                    <a href="/resource/academics" class="navigation-item"><span class="navigation-item-text">Academics</span></a>
-                </li>
-                <li class="pull-left">
-                    <a href="/resource/finances" class="navigation-item"><span class="navigation-item-text">Finances</span></a>
-                </li>
-                <li class="pull-left">
-                    <a href="/resource/studentcampuslife" class="navigation-item"><span class="navigation-item-text">Student and Campus Life</span></a>
-                </li>
-                <li class="pull-left">
-                    <a href="/resource/eventsactivities" class="navigation-item"><span class="navigation-item-text">Events &amp; Activities</span></a>
-                </li>
-                <li class="pull-left">
-                    <a href="/resource/toolssoftware" class="navigation-item"><span class="navigation-item-text">Tools &amp; Software</span></a>
-                </li>
-                <li class="pull-left">
-                    <a href="/resource/employment" class="navigation-item"><span class="navigation-item-text">Employment</span></a>
-                </li>
-            </ul>
         </div>
 
-        <div class="footer-bar" role="contentinfo">
-            <div class="container footer-links" style="position:relative;">
-                <ul>
-                    <li><a href="https://myuw.washington.edu/cgi-bin/contactmyuw.cgi"><i class="icon-envelope"></i> Contact</a></li>
-                     <li><a href="http://www.washington.edu/online/terms/">Terms</a></li>
-                     <li style="border:none;"><a href="http://www.washington.edu/online/privacy/">Privacy</a></li>
-                     <li style="border:none; position:absolute; right:0; top:50%; margin-top:-6px;"><a href="http://myuw.uw.edu"><i class="fa fa-desktop" aria-hidden="true"></i> Desktop Version</a></li>
-                </ul>
-                <div>&copy; {% now "Y" %} University of Washington</div>
+        <!-- footer -->
+        <div id="app_footer">
+
+            <div class="footer-bar" role="contentinfo">
+                <div class="container footer-links" style="position:relative;">
+                    <ul>
+                        <li><a href="https://myuw.washington.edu/cgi-bin/contactmyuw.cgi"><i class="icon-envelope"></i> Contact</a></li>
+                         <li><a href="http://www.washington.edu/online/terms/">Terms</a></li>
+                         <li style="border:none;"><a href="http://www.washington.edu/online/privacy/">Privacy</a></li>
+                    </ul>
+                    <div>&copy; {% now "Y" %} University of Washington</div>
```

Comment:
Ahhh! Good catch. I had removed all the {% static %} stuff, but missed this one.